### ISSUE TITLE
Urgent / PasswordTextFormField and PasswordTextField / missing style and strutStyle

### ISSUE THREAD

**Issue by vptcnt** [koji-1009/password_text_field#1]
Hello,

Could you implement the parameter style and strutStyle for PasswordTextFormField and PasswordTextField - the same as TextFormField and TextField.

It will allow giving the same look to all fields

```dart
TextFormField(
  //  ...
  style: style,
  strutStyle: strutStyle,
  // ...
)
TextField(
  //  ...
  style: style,
  strutStyle: strutStyle,
  // ...
)
```

Will be

```dart
PasswordTextFormField(
  //  ...
  style: style,
  strutStyle: strutStyle,
  // ...
)
PasswordTextField(
  //  ...
  style: style,
  strutStyle: strutStyle,
  // ...
)
```

Thanks a lot for your package...

**Comment by koji-1009** [comment]
Hi @vptcnt ,

Thanks your request.
I published v1.0.1!

https://pub.dev/packages/password_text_field/versions/1.0.1

**Comment by vptcnt** [comment]
迅速な対応に感謝
